includes/wf_crm_webform_preprocess.inc

```diff
@@ -784,17 +776,7 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
       }
     }
     if ($items) {
-      // Todo: Remove when we drop support for 4.4 - https://www.drupal.org/node/2832331
-      if (version_compare($this->civicrm_version, '4.6.0', '<')) {
```

Comment:
Remove when we drop support for 4.4 - about time! Thank you for the summer clean up and for also porting it to our D8WFC.